docs/Projects/TAD/ADRs/0003-tad-tool.md

```diff
@@ -0,0 +1,52 @@
+# ADR-0003 Tool for Transparency of Algorithmic Decision making
+
+## Context
+
+We are considering tooling for organizations to get more grip on their algorithms.
+Tooling for, for instance bias and fairness tests, and assessments (like [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes)).
+
+Transparency, we think, can be fostered by sharing reports from such a tool in a [standardized way](../Reporting-Standard/latest.md).
+
+There are several existing open source
+[tools](../Existing-Tools/Comparison/tools/) which
+we [have assessed](../Existing-Tools/Comparison/tools_comparison/).
+Some support only assessments, others already combine more features and can generate a report.
+There is however no tool
+that supports all the [requirements](../Existing-Tools/Comparison/requirements/) we have.
+
+These are our main requirements:
```

Comment:
In the requirements we refer to `project`, but in our case it's a project in the context of developing an algorithm. Or Algorithm System. What do we think of replacing project by something like `Algorithm System`?

Reply:
Tricky. I think project can apply to both situations, but do agreed the word project is not ideal. Then again, I don't think Algoritm System is the best replacement. I would suggested renaming the sentences containing 'the project' to something more general, but the requirements is also 'copied' to all the checklists so it is quite a refactor (though search/replace could solve that). So, I'll leave this for you and @anneschuth to reflect upon :)

Reply:
@uittenbroekrobbert let's not make this a merge blocker. In the context of the AI Validation Team people can get an idea about which kind of projects we are talking about.